pkg/sql/parser/sql.y

```diff
@@ -1372,19 +1372,19 @@ alter_table_cmd:
   {
     $$.val = &tree.AlterTableDropColumn{
       ColumnKeyword: $2.bool(),
-      IfExists: true,
```

Comment:
More than half of these were already aligned as if it were using gofmt.  So that's why I made the change.  We should have a consistent style and we don't.  I  opted for the style that matches the rest of our codebase.

Manually adding the spaces ins't a big deal but without linter support, it will not be enforceable.

There IS yacc file support in a lot of editors, but our specific blend of go and the grammar broke every one that I tried.

Reply:
> More than half of these were already aligned as if it were using gofmt. 

I don't see that's true. Can you show me?

Reply:
Just went back through and did a bit of an audit of them.  Most were not formatted.  But there is a mix.

Take a look at the `$$.val = &tree.CreateIndex{` Of the 4, 2 were formatted, 1 was not, and 1 was half formatted.

I'd still rather we format them in gofmt style to match the rest of our codebase.  But I'll leave the final decision up to you.  I find it more readable and I don't think the manual alignment is hard to do or enforce.  But the most important thing here is to be consistent.  So let me know either way.